utils/quectel_cm_5G/Makefile

```diff
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:= quectel-CM-5G
```

Comment:
It clearly duplicates `quectel-cm` package.

The whole thing is a mess IMHO.

Reply:
Oh sorry about that. i thought quectel-cm and quectel_cm_5g are different packages.